Makefile

```diff
@@ -16,41 +16,43 @@ else
 	endif
 endif
 
+DEFINES := CTEST_OUTPUT_ON_FAILURE=1
```

Comment:
Nice. This should be handy... I've been using the logs it stores in `build/darwin/Testing/Temporary/LastTest.log`.